content/en/docs/tasks/administer-cluster/namespaces.md

```diff
@@ -189,7 +189,8 @@ This delete is asynchronous, so for a time you will see the namespace in the `Te
     To demonstrate this, let's spin up a simple Deployment and Pods in the `development` namespace.
 
     ```shell
-    kubectl run snowflake --image=k8s.gcr.io/serve_hostname --replicas=2 -n=development
+    kubectl create deploy snowflake --image=k8s.gcr.io/serve_hostname  -n=development
+    kubectl scale deploy snowflake --replicas=2 -n=development
```

Comment:
Because this is documentation and is aimed at people who are relatively new to Kubernetes, I'd prefer to avoid the shorthand:
```suggestion
    kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname  -n=development
    kubectl scale deployment snowflake --replicas=2 -n=development
```